test/utils/utils.go

```diff
@@ -16,7 +17,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apimachinery "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
(optional) I think `k8serrors` is clearer personally.

```suggestion
	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```